cfg/admin.txt

```diff
@@ -13,3 +13,4 @@ giacomand
 rockdtben
 MrStonedOne
 Jordie0608
+Swankcookie
```

Comment:
I think you've got the wrong file here; this is for enabling access to client-side debugging and profiling tools.
/config/admins.txt is what's used for the admin tools.